meta/main.yml

```diff
@@ -15,5 +15,6 @@ galaxy_info:
     - "2017.09"
     - "2017.12"
```

Comment:
2018.03 here too?

Reply:
Same as above, I think we should only leave Karoo there. Thanks for bringing this up.